crates/api/src/runtime.rs

```diff
@@ -112,6 +115,7 @@ pub struct Store {
     context: Context,
     global_exports: Rc<RefCell<HashMap<String, Option<wasmtime_runtime::Export>>>>,
     signature_cache: HashMap<wasmtime_runtime::VMSharedSignatureIndex, ir::Signature>,
+    trap_registration_guards: Vec<TrapRegistrationGuard>,
 }
```

Comment:
Perhaps the guard should be with the `TrampolineState` along with the code memory?

Reply:
duh, attached to wrong CodeMemory, thanks